stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -117,3 +121,9 @@ spec:
         - name: docker-logs
           hostPath:
             path: /var/lib/docker
+      {{- if .Values.daemonset.tolerations }}
+      tolerations:
+{{ toYaml .Values.daemonset.tolerations | indent 8 }}
+      {{- end }}
+  updateStrategy:
+    type: {{ default "OnDelete" .Values.daemonset.updateStrategy | quote }}
```

Comment:
See comment on `values.yaml`, then `type: "{{ .Values.updateStrategy }}"` is sufficient